sensors_description/urdfs/sensors/zed_mini.gazebo.xacro

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0"?>
+
+<!--
+License: Apache 2.0. See LICENSE file in root directory.
+Copyright(c) 2017 PAL Robotics, S.L. All Rights Reserved
+
+This is the Gazebo URDF model for the Intel RealSense D435 camera
+-->
+
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+
+  <xacro:macro name="gazebo_zedm" params="camera_name reference_frame baseline left_depth_optical_frame 
+  right_depth_optical_frame left_color_optical_frame right_color_optical_frame 
+  gyro_frame camera_center odom_frame odom_frame_joint robot_name
+  disable_lumping:=true" >
+
+    <!-- Load parameters to model's main link-->
+    <xacro:property name="deg_to_rad" value="0.01745329251994329577" />
+
+    <!-- LEFT CAMERA WITH DEPTH -->
+    <gazebo reference="${left_color_optical_frame}">
+      <sensor name="left_camera" type="depth">
+        <update_rate>30</update_rate>
+        <camera>
+          <horizontal_fov>${90*deg_to_rad}</horizontal_fov>
+          <vertical_fov>${60*deg_to_rad}</vertical_fov>
+          <image>
+            <width>1920</width>
+            <height>1080</height>
+            <format>R8G8B8</format>
+          </image>
+          <clip>
+            <near>0.05</near>
+            <far>100.0</far>
+          </clip>
+        </camera>
+        <plugin name="depth_camera_controller" filename="libgazebo_ros_openni_kinect.so">
+          <baseline>0.2</baseline>
+          <alwaysOn>true</alwaysOn>
+          <updateRate>30.0</updateRate>
+          <cameraName>${camera_name}/left</cameraName>
+          <robotNamespace>${robot_name}</robotNamespace>
+          <imageTopicName>color/image_raw</imageTopicName>
+          <cameraInfoTopicName>color/camera_info</cameraInfoTopicName>
+          <depthImageTopicName>depth/image_raw</depthImageTopicName>
+          <depthImageInfoTopicName>depth/camera_info</depthImageInfoTopicName>
+          <pointCloudTopicName>depth/points</pointCloudTopicName>
+          <frameName>${left_depth_optical_frame}</frameName>
+          <pointCloudCutoff>0.3</pointCloudCutoff>
+          <pointCloudCutoffMax>15.0</pointCloudCutoffMax>
+          <!-- TODO modify intrinsic params -->
+          <distortionK1>0.00000001</distortionK1>
+          <distortionK2>0.00000001</distortionK2>
+          <distortionK3>0.00000001</distortionK3>
+          <distortionT1>0.00000001</distortionT1>
+          <distortionT2>0.00000001</distortionT2>
```

Comment:
If these params are specific for each camera, does it make sense to export them in the macro and leave some default values?

Reply:
Yes it make sense but I would wait the arrival of the real sensors and we can then associate the intrinsic parameters of that cameras here. Probably it would be better to open an issue.